README.md

```diff
@@ -17,8 +27,57 @@ For more information on Fixtures, check out this [pytest documentation](https://
 A **Mock** is an object that simulates the behavior of another object such as a class or function. They are used to simulate the behavior of real objects for testing purposes.
 For more information on Mocks, check out this [unittest documentation](https://docs.python.org/3/library/unittest.mock.html) on the mock module.
 
-**Example:**
+## Installation
+
+To use the **JupyterHub Pytest Plugin**, you will first need to install it using pip by either:
+
+- installing it locally
+  ```bash
+  pip install --upgrade pip
+  pip install git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
+  ```
+- adding it to the `requirements.txt` file of your project:
+
+  ```python
+  # in requirements.txt
+  git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
+  ```
+
+  then run:
+
+  ```bash
+  pip install --upgrade pip
+  pip install -r requirements.txt
+  ```
```

Comment:
When installing a package with `pip install git+https://github.com/jupyterhub/pytest-jupyterhub.git@main`, its dependencies get installed also.

Plus, we're defining the `pytest-jupyterhub`'s deps [in `pyproject.toml`](https://github.com/jupyterhub/pytest-jupyterhub/blob/43a862d0f19e07889c96ff7c48b26effcfff325d/pyproject.toml#L28-L29) and don't actually have a `requirements.txt` file in this repo, right?

Reply:
Oh yes! That is my mistake. All dependencies are defined in `pyproject.toml`. No `requirements.txt` file for this project, except for the docs.